src/archive.rs

```diff
@@ -53,10 +53,15 @@ impl CompressionMethod {
         }
     }
 
-    pub fn is_enabled(self, tar_enabled: bool, zip_enabled: bool) -> bool {
+    pub fn is_enabled(
+        self,
+        tar_enabled: bool,
```

Comment:
Let's go with `tar_enabled` and `tar_compressed_enabled`.

Reply:
Yes, it is now changed.